app/services/hot_lunch/sum_child_cost.rb

```diff
@@ -8,12 +13,33 @@ def call
     self.cost_adjustments = child.cost_adjustments
   end
 
+  def hot_lunch_charges
+    if child.hot_lunch_weeks.any?
```

Comment:
Save a level of indentation with a guard clause?
```suggestion
    return Money.empty if !child.hot_lunch_weeks.any?
```

Reply:
Yah that's cleaner